database/tbcd/level/upgrade.go

```diff
@@ -8,10 +8,246 @@ import (
 	"context"
 	"encoding/binary"
 	"fmt"
+	"os"
+	"path/filepath"
+	"sort"
+	"time"
+
+	"github.com/dustin/go-humanize"
+	"github.com/mitchellh/go-homedir"
+	cp "github.com/otiai10/copy"
+	"github.com/syndtr/goleveldb/leveldb"
+	"github.com/syndtr/goleveldb/leveldb/util"
+	"golang.org/x/sys/unix"
 
 	"github.com/hemilabs/heminetwork/database/level"
+	"github.com/hemilabs/heminetwork/rawdb"
+)
+
+var (
+	upgradeVerbose = true
+	batchSize      = 1_000_000     // move one million records per batch
+	chunkSize      = 1_000_000_000 // 1GB
+	gib            = uint64(1024 * 1024 * 1024)
+
+	modeMove = true
+
+	modeFast = true // try to run fast when enough disk space is availale
 )
 
+func SetMode(move bool) {
+	modeMove = move
+}
+
+func SetBatchSize(size int) {
+	batchSize = size
+}
```

Comment:
As these are exported in the `level` package, would it make sense to add "Upgrade" to them to prevent them from being confused with being "standard" database functions?

```suggestion
func SetUpgradeMode(move bool) {
	modeMove = move
}

func SetUpgradeBatchSize(size int) {
	batchSize = size
}
```

Reply:
Yes, we need those.